src/UserOuput/UserOutput.js

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
-import './UserOutput.css'
+import './UserOutput.css';
+import ValidationComponent from '../ValidationComponent/ValidationComponent';
+import CharComponent from '../CharComponent/CharComponent';
 
 function userOutput(props) {
+  let letters = null;
+
+  if (props.text.length) {
+    letters = props.text.split('').map((letter, index) =>
+      <CharComponent
+        key={index}
+        letter={letter}
+        onLetterClick={() => props.onLetterClick(index)}/>
+    )
+  }
+
   return (
     <div className="UserOutput">
-      <p>Some random text: </p>
-      <p>{props.username}</p>
+      <p className="userOutputDescription">User input: </p>
+      <p>Length: {props.text.length}</p>
```

Comment:
`props.text` used more then one time, destrcut it in the beginnig as
```
cosnt {text, username} = props
```

Reply:
done